google/cloud/forseti/scanner/audit/forwarding_rule_rules_engine.py

```diff
@@ -73,6 +73,11 @@ def find_policy_violations(self, forwarding_rule,
             self.build_rule_book()
         resource_rules = self.rule_book.get_resource_rules()
 
+        # If there is no forwarding rules defined in the rule file then no
+        # forwarding rule is violated.
+        if not resource_rules:
```

Comment:
I am just wondering if it would be more efficient to have this check upstream, where the rule book is built, and raise an exception so that we can save some cycle like this.  Okay as is, but please put a TODO for the future.

Reply:
Updated.